src/cmd/CMakeLists.txt

```diff
@@ -14,6 +14,11 @@ if(NOT BUILD_LIBRARIES_ONLY)
 
     # Need tokenizer for netrc parser
     target_include_directories(${cmd_NAME} PRIVATE ${CMAKE_SOURCE_DIR}/src/3rdparty/qtokenizer)
-    install(TARGETS ${cmd_NAME} ${KDE_INSTALL_TARGETS_DEFAULT_ARGS})
+
+    if(APPLE)
+      install(TARGETS ${cmd_NAME} RUNTIME DESTINATION "${KDE_INSTALL_BUNDLEDIR}/${APPLICATION_EXECUTABLE}.app/Contents/MacOS/")
```

Comment:
For the shell extension I cam up with https://github.com/owncloud/client/blob/master/shell_integration/MacOSX/CMakeLists.txt#L18
I think the generator expression is nicer, I can't remember why I decided to use the copy command but I think I had issues with the install command....